inc/Media.php

```diff
@@ -48,9 +48,10 @@ function svg($path, $args = []) {
  *
  * @param mixed $image
  * @param string $size
- * @param boolean $responsive
+ * @param string $class
+ * @param string $sizes
  */
-function image($image, $size = 'medium', $responsive = true) {
+function image($image, $size = 'medium', $class = null, $sizes = null) {
```

Comment:
So I'm not a fan of removing the $responsive parameter. Sometimes you don't want responsive images. Either way, "the golden unwritten rule" for function parameters is to never go over 3 parameters, and instead use an options array. You can use `params` function for setting defaults in the options array.

Reply:
After thinking about this some more, I'm now agreeing with keeping the $responsive -parameter. For example if we are showing the thumbnail image in some link of listing, we probably do not need any responsiveness to it at all. I will refactor this to work with the options-array.

Reply:
This is now refactored to use options-array. Should we have the size as a direct parameter to the function instead of adding it to the options-array? As in:

```suggestion
function image($image, $size = 'medium', $args) {}
```

Reply:
Your call, I'd probably keep the $size in it's own parameter.

Reply:
I agree with that, makes the usage a bit faster when going directly with the defaults.